packages/core/test/stylable-transformer/scoping.spec.ts

```diff
@@ -799,6 +799,36 @@ describe('Stylable postcss transform (Scoping)', () => {
 
             testInlineExpects(result);
         });
+
+        it('should scope multiple pseudo-elements that extended none root anchor', () => {
```

Comment:
this test is redundant in my opinion. You already checked the transform context target flow in the other test. it is not specifically related to pseudo-elements

Reply:
the fix is for the transformation flow of pseudo-classes, the only flow that calls `scopeSelectorAst` with a nested context is a functional pseudo-class